File: KoRadio/KoRadio.Model/Company.cs
using System;
using System.Collections.Generic;

namespace KoRadio.Model
{



    public partial class Company
    {
		public int CompanyId { get; set; }
		public string CompanyName { get; set; } = null!;
		public string Email { get; set; } = null!;

		public string Bio { get; set; } = null!;

		public decimal? Rating { get; set; }

		public string PhoneNumber { get; set; } = null!;

		public int ExperianceYears { get; set; }

		public byte[] Image { get; set; } = null!;

		public List<DayOfWeek> WorkingDays { get; set; }

		public TimeOnly StartTime { get; set; }

		public TimeOnly EndTime { get; set; }

		public int LocationId { get; set; }

		public bool IsDeleted { get; set; }
		public bool IsApplicant { get; set; }


		public virtual ICollection<CompanyService> CompanyServices { get; set; } = new List<CompanyService>();
		public virtual ICollection<CompanyEmployee> CompanyEmployees { get; set; } = new List<CompanyEmployee>();

		public virtual Location Location { get; set; } = null!;
	}
}

File: KoRadio/UI/ko_radio_mobile/lib/models/company.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:ko_radio_mobile/models/company_employee.dart';
import 'package:ko_radio_mobile/models/company_services.dart';
import 'package:ko_radio_mobile/models/location.dart';

part 'company.g.dart';
@JsonSerializable()
class Company {
  int companyId;
  String companyName;
  String email;
  String bio;
  double rating;
  String phoneNumber;
  int experianceYears;
  String? image;
  List<String>? workingDays;
  String startTime;
  String endTime;
  int? locationId;
  Location? location;
  bool isDeleted;
  bool isApplicant;
  List<CompanyEmployee>? companyEmployees;
  List<CompanyServices> companyServices; 
  Company(this.companyId ,this.companyName, this.bio, this.rating,
   this.phoneNumber, this.experianceYears, this.image, this.workingDays, this.startTime, 
   this.endTime, this.location, this.isDeleted, this.isApplicant, this.companyEmployees, this.companyServices,this.email);

  factory Company.fromJson(Map<String, dynamic> json) => _$CompanyFromJson(json);
  Map<String, dynamic> toJson() => _$CompanyToJson(this);
}